tgui/packages/tgui/interfaces/OreRedemption.js

```diff
@@ -0,0 +1,229 @@
+import { useBackend } from '../backend';
+import { Box, Button, Divider, Flex, Icon, LabeledList, NumberInput, Section } from '../components';
+import { Window } from '../layouts';
+
+const formatPoints = amt => amt.toLocaleString("en-US") + " pts";
+
+const iconNameOverrides = {
+  bananium: "clown",
+  tranquillite: "mime",
+};
+
+export const OreRedemption = (properties, context) => {
+  return (
+    <Window>
+      <Window.Content>
+        <Flex
+          direction="column"
+          width="100%"
+          height="100%">
+          <Flex.Item basis="content" mb="0.5rem">
+            <IdDisk height="100%" />
+          </Flex.Item>
+          <Flex.Item grow="1" overflow="hidden">
+            <Ores height="100%" />
+          </Flex.Item>
+        </Flex>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const IdDisk = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    id,
+    points,
+    disk,
+  } = data;
+  const {
+    ...rest
+  } = properties;
+  return (
+    <Section {...properties.rest}>
```

Comment:
Ima be stupid here a second. Didnt you just create `rest` in the lines above this? And if so, why do you need to prefix with `...properties`

Reply:
oof